systems/init/lookup/internal/server/lookup.go

```diff
@@ -242,7 +241,7 @@ func (l *LookupServer) GetNode(ctx context.Context, req *pb.GetNodeRequest) (*pb
 }
 
 func (l *LookupServer) GetNodeForOrg(ctx context.Context, req *pb.GetNodeForOrgRequest) (*pb.GetNodeResponse, error) {
-	logrus.Infof("Get node %s for org %s.", req.GetNodeId(), req.GetOrgName())
+	log.Infof("Get node %s for org %s.", req.GetNodeId(), req.GetOrgName())
 
 	_, err := l.orgRepo.GetByName(req.OrgName)
```

Comment:
What’s the purpose of this  	`_, err := l.orgRepo.GetByName(req.OrgName) `if the return value isn’t being used?